js/popup.js

```diff
@@ -0,0 +1,304 @@
+      altText: 'project 6 preview image',
+    },
+    name: 'Multi-Post Stories Gain+Glory',
+    heading: 'Keeping track of hundreds of components',
+    languages: [
+      'Ruby on Rails',
+      'Css',
+      'JavaScript',
+      'html',
+    ],
+    technologies: [
+      'Codekit',
+      'GitHub',
+      'Javascript',
+      'Bootstrap',
+      'Terminal',
+      'Codepen',
+    ],
+    // button: 'See Project',
+    description: 'Project_6 Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    list: [
+      {
+        text: 'See Live',
+        icon: './assets/icons/Icon-Export@2x.svg',
+      },
+      {
+        text: 'See Source',
+        icon: './assets/icons/GitHub.svg',
+      },
+    ],
+  },
+];
+
+function createCard(cardObject) {
+  const li = document.createElement('li');
+  li.className = 'card--section';
+  li.innerHTML = `<div class="image--container">
+  <img src="${cardObject.images.img}" alt="${cardObject.images.altText}">
+  </div>
+  <div class="btn--container">
+    <h4>${cardObject.name}</h4>
+    <ul class="ul--section">
+        ${cardObject.languages.map((lang) => `<button type="button" class="btn">
+        ${lang}</button>`).join('')}
+    </ul>
+  </div>
+  <div class="see--project">
+    <button type="button" class="btn--style ${cardObject.id}">See Project</button>
+  </div>`;
+
+  return li;
+}
+
+projects.forEach((project) => {
+  const card = createCard(project);
+  mainWorkSection.appendChild(card);
+});
+
+function createPopup(object) {
+  const popupCard = document.createElement('div');
+  popupCard.classList.add('card-popup');
+  popupCard.id = object.id;
+  popupCard.innerHTML = `
+  <div class="project-img">
+    <img class="card-img" src="${object.images.img}" alt="${object.images.altText}">
+    <img class="hide-icon" src="./assets/icons/Disabled@2x.png"></div>
+  
+  <div class="article">
+    <h4>${object.heading}</h4>
+    <ul class="ul--section d-flex">
+        ${object.languages.map((lang) => `<button class="btn mobile-invisible"> 
+        ${lang}</button>`).join('')}
+
+        ${object.technologies.map((tech) => `<button class="btn desktop-invisible"> 
+        ${tech}</button>`).join('')}
+
+    </ul>
+    <p>${object.description}</p>
+    <div class="see--live">
+      <button type="button" class="btn--style"><span>See Live</span><img src="./assets/icons/Icon-Export@2x.svg" alt=""></button>
+      <button type="button" class="btn--style"><span>See Source</span><img src="./assets/icons/GitHub.svg" alt=""></button>
+    </div>
```

Comment:
- [ ] I think it would be nice to have a matching alignment and spacing of elements with the design given, currently `see live` and `see source` buttons are misaligned in desktop view as shown below:
#### Your project 
![image](https://user-images.githubusercontent.com/61377322/146307401-8290d80e-91d0-44a0-b64e-da121a3b3985.png)

#### Figma design
![image](https://user-images.githubusercontent.com/61377322/146307411-e7bbe59b-2f05-42a5-8bb3-d8e61be02e0c.png)